src/app/test-record/test-record.mapper.ts

```diff
@@ -29,6 +34,23 @@ export class TestRecordMapper {
     }
   }
 
+  getReasonsForAbandoning(vehicleType: string, testTypeId: string) {
```

Comment:
Can we test this function please?
If we plan to reuse tirTestTypeIds anywhere else we could move them into a constant file or where we declared the test-record enums